libraries/nomad_job.rb

```diff
@@ -20,16 +21,40 @@ def initialize(url, job_name)
   end
 
   def running?
-    job = http_json("#{@url}/v1/job/#{@job_name}")
+    job = get_job
```

Comment:
```suggestion
    job_status == 'running'
```
Should work as well?